kong/tools/utils.lua

```diff
@@ -291,6 +30,7 @@ do
   local KEY_PTRN  = "(?!kong)(?!konnect)(?!insomnia)(?!mesh)(?!kic)" .. BASE_PTRN
   local VAL_PTRN  = BASE_PTRN
 
+
```

Comment:
should not add this blank line, we should keep the original code.

Reply:
done